csharp/OpenMatch/OpenMatch.csproj

```diff
@@ -11,6 +11,8 @@
 
   <ItemGroup>
     <PackageReference Include="Google.Api.CommonProtos" Version="1.7.0" />
+    <PackageReference Include="Google.Protobuf" Version="3.8.0" />
+    <PackageReference Include="Grpc" Version="2.26.0" />

```

Comment:
When I did this locally I only needed the `Grpc` package for the build to pass; ymmv. :)

Reply:
I see. I'm following the guide in this [link](https://docs.microsoft.com/en-us/aspnet/core/grpc/basics?view=aspnetcore-3.1#c-tooling-support-for-proto-files).